### DISCUSSION TITLE
laravel 10, PHP 8下无法安装，5.0，6.0都试过了

### DISCUSSION THREAD

**Discussion by brianping7** [w7corp/easywechat#2756] [General]
Problem 1
    - overtrue/laravel-wechat[5.0.0, ..., 5.0.1] require illuminate/container ^5.1 -> found illuminate/container[v5.1.1, ..., v5.8.36] but these were not loaded, likely because it conflicts with another require.
    - overtrue/laravel-wechat[5.0.2, ..., 5.0.3] require illuminate/container ^5.1||~6.0 -> found illuminate/container[v5.1.1, ..., v5.8.36, v6.0.0, ..., v6.20.44] but these were not loaded, likely because it conflicts with another require.
    - overtrue/laravel-wechat[5.0.4, ..., 5.0.7] require illuminate/container ^5.1 || ^6.0 || ^7.0 -> found illuminate/container[v5.1.1, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6] but these were not loaded, likely because it conflicts with another require.
    - overtrue/laravel-wechat[5.0.8, ..., 5.1.0] require illuminate/container ^5.1 || ^6.0 || ^7.0 || ^8.0 -> found illuminate/container[v5.1.1, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires overtrue/laravel-wechat ^5.0 -> satisfiable by overtrue/laravel-wechat[5.0.0, ..., 5.1.0].
有人遇到过嘛

**Comment by overtrue** [comment]
1. 请使用 packagist 官方源，盲猜你用了阿里已经不维护的源。
2. 最新的 laravel-wechat 是 7.2.0, 你的报错是5.0.0，安装最新版： `composer require overtrue/laravel-wechat:^7.2`
